src/main/kotlin/com/intershop/gradle/icm/docker/utils/AbstractTaskPreparer.kt

```diff
@@ -90,13 +95,17 @@ abstract class AbstractTaskPreparer(
         task.group = "icm container ${getContainerExt()}"
         task.attachStderr.set(true)
         task.attachStdout.set(true)
-        task.hostConfig.autoRemove.set(true)
+        //task.hostConfig.autoRemove.set(true)
```

Comment:
Could this task configuration be made configurable (with default value `true` as before)?

Reply:
Could be but would double the effort. But I uncommented the line so auto-remove is now enabled (-> proper cleanup)-